x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/count.tsx

```diff
@@ -23,21 +24,31 @@ export const countOperation: OperationDefinition<CountIndexPatternColumn> = {
   displayName: i18n.translate('xpack.lens.indexPattern.count', {
     defaultMessage: 'Count',
   }),
-  getPossibleOperationForDocument: () => {
+  onFieldChange: (oldColumn, indexPattern, field) => {
```

Comment:
Why make this change?

Reply:
Same reason as above. We now have a single kind of operation, rather than a special-case for documents.